README.md

```diff
@@ -1,22 +1,67 @@
 # Grid Weblets
 
-![Version: 1.0.0](https://img.shields.io/github/v/release/threefoldtech/grid_weblets?style=flat-square)
+![Version: 1.0.0](https://img.shields.io/github/v/release/threefoldtech/grid_weblets)
+[![CI-CD](https://github.com/threefoldtech/grid_weblets/actions/workflows/ci-cd.yml/badge.svg)](https://github.com/threefoldtech/grid_weblets/actions/workflows/ci-cd.yml)
 
-A svelte project for creating web components - which are reusable custom elements with their functionality encapsulated away from the rest of the code — that interact with TF Grid 3 and could be utilized from other web apps.
+## Introduction
+
+Weblets is a front-end application helps to deploy solutions on TF Grid v3. A weblet is a compiled javascript web component which can be embedded in HTML page of a web app. The backend for the weblets is introduced with [grid client](https://github.com/threefoldfoundation/info_manual3/blob/development/wiki/manual3_iac/grid3_javascript/grid3_javascript_home.md) which communicate to TF Chain and TF Grid over RMB.
+
+Weblets is a svelte project for creating web components - which are reusable custom elements with their functionality encapsulated away from the rest of the code — that interact with TF Grid 3 and could be utilized from other web apps.
 It solves such problem where you had to write complex HTML (and associated style and script) to render custom UI controls (eg, to deploy some workload on the TF Grid 3), and how using them multiple times in different projects can be a miss if you are not careful.
 
-To Try the project locally you can easy clone the repo and run the following commands:
+## Installation
+
+- **Get the code**
+
+  Clone the repository:
+
+  ```bash
+  git clone https://github.com/threefoldtech/grid_weblets.git
+  ```
+
+- [**Prepare the environment**](./docs/config.md)
+
+## Getting Started
+
+> For detailed information you can read the [Getting Started](./docs/getting_started.md) documentation.
+
+Before run Weblets, in your terminal move to the project directory.
+
+- **Run Weblets in Development**
 
 ```bash
-yarn deps
 yarn serve:app
 ```
 
-Read the documentaions for more information.
+- **Run Weblets in Production**
+
+```bash
+yarn build:app
+```
+
+## Configure the editor/IDE
```

Comment:
Move this section to a new md file, and add development setup section mentioning all the configs to those extensions(If needed)

Reply:
please check: https://github.com/threefoldtech/grid_weblets/blob/development_update_docs/docs/editor_config.md